### PR TITLE
Expand/simplify example: getStorage with keys definitions

### PR DESCRIPTION
**Pull request by jacogr**
Closes https://github.com/polkadot-js/apps/issues/136

First step... we still should not have to do the publicKey -> ss58 encoding, https://github.com/polkadot-js/api/issues/109

It really does not go far enough (yet) to reduce the boilerplate required. But it is a step.

### CODE REVIEW

File: packages/app-example/src/comp-004.tsx [hunk 1]
```diff
@@ -45,39 +46,34 @@ class Comp extends React.PureComponent<ApiProps, State> {
...
-    const transform = storageTransform(intentionsMethod);
 
     return api.state
-      .getStorage(key)
-      .subscribe((value) => {
+      .getStorage(storage.staking.public.intentions)
+      .subscribe((intentions: StorageIntentions) => {
         this.setState({
-          intentions: (transform(value, 0) as any[]).map(encodeAddress)
+          intentions: intentions.map(encodeAddress)
```
Comment (jacogr) [inline packages/app-example/src/comp-004.tsx]: This needs to be dropped next https://github.com/polkadot-js/api/issues/109

**Comment by ltfschoen** [comment]
* back-end - built the latest Polkadot Node master branch and ran the following in Dev mode:

      ```
      rm -rf /tmp/alice*/;
      ./target/debug/polkadot --chain=dev --validator --key Alice -d /tmp/alice
      ```

* front-end

      ```
      git fetch upstream pull/137/head:jg-example-simplify;
      git checkout jg-example-simplify;
      yarn install;
      yarn run start;
      ```

    * Open http://localhost:3000/#/example, click Tut004, shows 1 person staking (i.e. Alice)
    * Generate the JSONRPC calls by copy/paste the following cURL command in terminal (instead of having to click through the Extrinsics tab in the UI again) to create an Extrinsic JSONRPC call `author_submitExtrinsic`, which calls the `propose` or `stake` function in Polkadot Client codebase: substrate > runtime
    * Charlie > staking > stake() - 

```
curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"author_submitExtrinsic","params": ["0x67000000ffa39bffaf38151046073703e9756f8ce49eeba0534151f7e3fbc6aa185f06b3a400000000020187b2f9f664c902f7e54ef894cfee9a8faa2572deea581c415750a1771c2033655f57eb312cfc31f54f6a290d44e1843a78bfe5ca49d06c5641cc81ca8e5f6209"], "id":"33"}' 127.0.0.1:9933
```

    * Open http://localhost:3000/#/example, click Tut004, now shows 2 people staking (i.e. Alice, Charlie)
    * Charlie > democracy > propose > consensus > setCode(), (4kB text file ok, drag-drop 8 kB image 4x6 pixels crashes db)
      * Or just copy/paste the following in terminal

```
curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"author_submitExtrinsic","params": ["0x81000000ffa39bffaf38151046073703e9756f8ce49eeba0534151f7e3fbc6aa185f06b3a4010000000500000004000000307830300100000000000000000000000000000032dc2b6d7d29dd46c47cda6eb42f8cd1bc90436ad4a13fca4515394449b57afc1a821649b840ad4a1c41e12fbff3628a969ad7532f708630aa969aba6c3e1306"], "id":"30"}' 127.0.0.1:9933
``` 

    * Result: Open http://localhost:3000/#/example, click Tut004, still shows 2 people staking (i.e. Alice, Charlie), but still says 0 proposals (instead of 1)

* Note that if I try running two nodes in Local mode it isn't generating any blocks as it was earlier today for some reason
      
```
rm -rf /tmp/alice*/;
./target/debug/polkadot --chain=local --validator --key Alice -d /tmp/alice;
./target/debug/polkadot --chain=local --validator --key Bob --port 30334 -d /tmp/bob --bootnodes 'enode://ALICE_BOOTNODE_ID@127.0.0.1:30333'
```

**Comment by ltfschoen** [comment]
i tried again making sure i chose a value greater than the minimum amount of `9` that was shown in the Storage tab > democracy > minimumDeposit

**Comment by jacogr** [comment]
@ltfschoen Please log the above as an issue (copy is fine) - was not caused by this PR (nothing there touched, this only queries), but there is most def. something up somewhere.

**Pull request merged**